### ISSUE TITLE
Format docstrings

### ISSUE THREAD

**Issue by nathanshammah** [watermarkhu/OpenSurfaceSim#14]
The two most common docstring formats are Numpy-style and Google-style (more compact). They both contain standard keywords for arguments, returned objects and the possibility to annotate the signature with type hints or within the docstring itself. 

This is particularly helpful to automatically build the API-doc as part of the documentation with Sphinx's autodoc's feature. 

Numpydoc helps ensuring this happens, https://numpydoc.readthedocs.io/en/latest/format.html#.

Happy to help on this with contributions or feedback.

**Comment by watermarkhu** [comment]
@nathanshammah 
How does Sphinx handle inherited classes?  For example if I have a class 
````python
class foo:
"""Docstring of superclass

Notes
-----
This is the docstring of the superclass
"""
pass
````
and I inherit this class while also optionally add a docstring here

```python
class bar(foo):
"""Docstring of the inherited class
"""
pass
```
what does the documentation show for the inherited class?

**Comment by watermarkhu** [comment]
Implemented in #26
